packages/aragon-wrapper/src/utils/transactions.js

```diff
@@ -40,7 +40,7 @@ export async function createDirectTransaction (sender, app, methodName, params,
   }
 
   if (transactionOptions.token) {
-    const { address: tokenAddress, value: tokenValue } = transactionOptions.token
+    const { address: tokenAddress, value: tokenValue, spender } = transactionOptions.token
```

Comment:
We should add this to the documentation about the `token` parameter (we could also fix the link to the Finance app example): https://github.com/aragon/aragon.js/blob/master/docs/APP.md#aragonapp